### DISCUSSION TITLE
CRC 32

### DISCUSSION THREAD

**Discussion by Iceymidgit** [sanni/cartreader#720] [Need help]
I'm Pal based region ( Europe )  and some games i keep getting CRC32 ... numbers not found 

EG Super Mario Bros. + Duck Hunt (Europe).nes CRC 32 86722EB7 -> nor found

**Comment by sanni** [comment]
Those are bad dumps. Therefore the CRC32 can't be found. I would either clean the cartridges or re-solder the Cart Reader.

A telltale sign is when the name is not displayed correctly then dumping will not succeed. 
For Super international cricket the title should be: INTERNATIONAL.CRICKET (the dot might get replaced by a _ )and for Super Mario RPG ( japan ) it should be: SUPER MARIO RPG
